start.xenit.eu/src/main/java/eu/xenit/alfred/initializr/start/sdk/alfred/compose/config/ComposeUpGradleTaskConfiguration.java

```diff
@@ -25,15 +25,7 @@ public ComposeUpGradleTaskConfiguration(DockerImageEnvNameProvider variableNameP
     @Override
     public void accept(GradleTask.Builder builder) {
         this.usesDockerImageFrom.forEach(dockerImageProject -> {
-            String dockerImageEnvName = this.variableNameProvider.get(dockerImageProject);
-
             builder.invoke("dependsOn", "':"+dockerImageProject.getId()+":buildDockerImage'");
-            builder.nested("doFirst", doFirst -> {
```

Comment:
this only works with newer versions of the `eu.xenit.docker` but I don't think it's updated yet ?

Reply:
ah, it is - never mind